index.d.ts

```diff
@@ -73,7 +74,58 @@ declare namespace Eris {
   type MessageContent = string | AdvancedMessageContent;
   type MFALevel = 0 | 1;
   type PossiblyUncachedMessage = Message | { channel: TextableChannel | { id: string; guild?: Uncached }; guildID?: string; id: string };
-  type InteractionType = 1 | 2;
+
+  // Interaction
+  type InteractionDataOptions = {
+    name: string;
+    type: Constants["ApplicationCommandOptionTypes"][keyof Constants["ApplicationCommandOptionTypes"]];
+    value?: string;
```

Comment:
Is this supposed to be a string or the type from line 82?

Reply:
90% sure it's the type

Reply:
yep https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-interaction-data-option-structure

Reply:
actually it says the value, but links to the type, im not sure, its mutually exclusive with options tho so i have no idea

Reply:
perhaps just change it to this?
```suggestion
    value?: string | number | boolean;
```

Reply:
Yea ill make it that until we prove otherwise